docs/users-guide/03-basic-exploration.md

```diff
@@ -69,6 +69,26 @@ Searches take into account items’ titles, descriptions, and other metadata —
 
 ![Search results](./images/basic-exploration/search-results.png)
 
+## Bookmarks
+
+**Bookmarks** are a way to quickly get back to stuff you visit frequently (or have been working on recently). Bookmarked items show up in the main, left sidebar above [collections][collections].
+
+To bookmark an item, look for the ribbon icon. If you don't immediately see the ribbon, open up the settings for that item. You can bookmark:
```

Comment:
Incidentally, Agnes is exploring ways to unify where this action lives on all entities for a future release. It's a bit all over the place for v1.